crates/proto/src/gen/penumbra.core.transparent_proofs.v1alpha1.rs

```diff
@@ -1,25 +1,121 @@
+// @generated
 /// A Penumbra transparent SwapClaimProof.
 #[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct SwapClaimProof {
     /// The swap being claimed
-    #[prost(message, optional, tag = "1")]
-    pub swap_plaintext: ::core::option::Option<
-        super::super::dex::v1alpha1::SwapPlaintext,
-    >,
+    #[prost(message, optional, tag="1")]
+    pub swap_plaintext: ::core::option::Option<super::super::dex::v1alpha1::SwapPlaintext>,
     /// Inclusion proof for the swap commitment
-    #[prost(message, optional, tag = "4")]
-    pub swap_commitment_proof: ::core::option::Option<
-        super::super::crypto::v1alpha1::StateCommitmentProof,
-    >,
+    #[prost(message, optional, tag="4")]
+    pub swap_commitment_proof: ::core::option::Option<super::super::crypto::v1alpha1::StateCommitmentProof>,
     /// The nullifier key used to derive the swap nullifier
-    #[prost(bytes = "vec", tag = "6")]
+    #[prost(bytes="vec", tag="6")]
     pub nk: ::prost::alloc::vec::Vec<u8>,
     /// *
     /// @exclude
     /// Describes output amounts
-    #[prost(message, optional, tag = "20")]
+    #[prost(message, optional, tag="20")]
     pub lambda_1_i: ::core::option::Option<super::super::crypto::v1alpha1::Amount>,
-    #[prost(message, optional, tag = "21")]
+    #[prost(message, optional, tag="21")]
     pub lambda_2_i: ::core::option::Option<super::super::crypto::v1alpha1::Amount>,
 }
+/// Encoded file descriptor set for the `penumbra.core.transparent_proofs.v1alpha1` package
+pub const FILE_DESCRIPTOR_SET: &[u8] = &[
+    0x0a, 0xe1, 0x0b, 0x0a, 0x42, 0x70, 0x65, 0x6e, 0x75, 0x6d, 0x62, 0x72, 0x61, 0x2f, 0x63, 0x6f,
+    0x72, 0x65, 0x2f, 0x74, 0x72, 0x61, 0x6e, 0x73, 0x70, 0x61, 0x72, 0x65, 0x6e, 0x74, 0x5f, 0x70,
+    0x72, 0x6f, 0x6f, 0x66, 0x73, 0x2f, 0x76, 0x31, 0x61, 0x6c, 0x70, 0x68, 0x61, 0x31, 0x2f, 0x74,
+    0x72, 0x61, 0x6e, 0x73, 0x70, 0x61, 0x72, 0x65, 0x6e, 0x74, 0x5f, 0x70, 0x72, 0x6f, 0x6f, 0x66,
```

Comment:
👀 Looks like the new setup is putting the file descriptor set in the output directly?